website: Update 'Contributing' page (#72)
gem5/new-website: _pages/contributing.md

The commit message must adhere to our style. The first line of the commit is
the "header". The header starts with a tag (or tags, separated by a comma),
then a colon. Which tags are used depend on which components of gem5
you have modified. **Please refer to the [TAGS.yaml](

> https://github.com/gem5/gem5/blob/stable/TAGS.yaml) for

Reviewer: Probably want to hold off on this for now.

In addition to the CI tests, your changes will be reviewed by the gem5 community.
Your pull-request must have the approval of at least one community member prior to being merged.

Once your pull-request has passed all the CI tests and has been approved by at least one community member, it will be merged a gem5 maintainer will do a [Merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges) on the pull-request.

> The gem5 maintainers are a small group of developers who have been granted the ability to merge pull-requests into the gem5 `develop` branch.

Reviewer: ```suggestion
The gem5 maintainers the individuals granted the ability to merge pull-requests into the gem5 `develop` branch.
```

When you understand what changes are required make amendments to the pull
request by adding patches to the same branch and then pushing to the forked repository.

> A git "force push" (i.e., `git push --force`) is also acceptable if you wish to alter the commits locally in order to make the changes.

Reviewer: Should we add that we would prefer if people did a rebase and/or amend and force pushed instead of applying changes to the end, if at all possible?

Once pushed to the forked repository, the pull request will automatically update with your changes.
The reviewer will then re-review your changes and, if necessary, ask for further changes, or approve your pull-request.

## Reviewing other contributions

We encourage all gem5 developers to review other's contributions.
Anyone may review a gem5 change and, if they feel it is ready, approve it.
All pull-requests can be found at <https://github.com/gem5/gem5/pulls>.

When reviewing a pull request we enforce the followings guidelines.
These have been designed to ensure clear and polite communication between all parties:

> 

Reviewer: Putting this link would be good, too: https://google.github.io/eng-practices/review/reviewer/comments.html

* In all forms of communication, contributors and reviewers must be polite.
Comments seen as being rude or dismissive will not be tolerated.

> * If choosing to not approve a PR, please state clearly why.

Reviewer: ```suggestion
* If choosing to not approve a PR, please state clearly why.
When asking for changes, the comments should be specific and actionable.
```
